src/include/sof/alloc.h

```diff
@@ -83,7 +83,7 @@ struct mm_info {
 struct block_hdr {
 	uint16_t size;		/* size in blocks for continuous allocation */
 	uint16_t used;		/* usage flags for page */
-} __attribute__ ((packed));
+} __packed;
```

Comment:
Idk if it's right to listen to checkpatch in this place. Isn't `__attribute__` more widely supported than `__packed`? But, since for not we care only about gcc-compatible compilers I guess it's not a big deal.

Reply:
Nvm I just now see below that it's defined in sof.h.